client/app/scripts/superdesk-publish/filters/view/filter-search.html

```diff
@@ -0,0 +1,28 @@
+<div>
+    <form name="searchForm" ng-submit="search()">
+        <fieldset>
+            <div class="field">
+                <label translate>Field</label>
+                <select class="input" ng-change="resetValues()" required ng-model="filterCondition.field">
+                    <option value="{{parameter.field}}" ng-selected="parameter.field === filterCondition.field" ng-repeat="parameter in filterConditionParameters">{{:: parameter.field}}</option>
+                </select>
+            </div>
+            <div class="field">
+                <label translate>Operator</label>
+                <select class="input" ng-change="resetValues()" required ng-model="filterCondition.operator" >
+                    <option value="{{operator}}" ng-selected="operator === filterCondition.operator" ng-repeat="operator in operatorLookup[filterCondition.field]">{{operator}}</option>
```

Comment:
Same thought (at two places).

We should probably add `track by` and `translate`?

Also, could the same be achieved with `ng-options` perhaps?